apps/graphql/src/definitions/Repository.js

```diff
@@ -170,21 +173,27 @@ export const resolvers = {
         return repository;
       });
     },
-    async updateBaselineBranch(source, { repositoryId, branchName }, context) {
+    async updateBaselineBranch(
+      source,
+      { repositoryId, baselineBranch = "" },
```

Comment:
```suggestion
      { repositoryId, baselineBranch },
```
It is required, so I don't get the default, how can it be undefined?